vendor/modules.txt

```diff
@@ -98,6 +98,7 @@ github.com/golang/protobuf/ptypes/any
 github.com/golang/protobuf/ptypes/duration
 github.com/golang/protobuf/ptypes/timestamp
 # github.com/google/uuid v1.3.0
+## explicit
```

Comment:
What do you mean by this comment?

Reply:
TBH not sure, it was added automatically, let me check.

Reply:
This is a related thread https://github.com/golang/go/issues/37946#issuecomment-601210447
I didn't understand the `explicit` comment completely, but I can see we already have a bunch of these comments in this file.